st2common/st2common/util/action_db.py

```diff
@@ -215,7 +215,6 @@ def get_args(action_parameters, action_db):
     for _, arg in six.iteritems(position_args_dict):
         positional_args.append(str(action_parameters.get(arg)))
         positional_args_keys.add(arg)
-    positional_args = ' '.join(positional_args)  # convert to string.
```

Comment:
We were converting pos args to a string here which is not right. We should pass it as a list and let ssh runners deal with it. I am also not happy that this method exists in this file. This is specific to script runners. Anyways, one step at a time.